includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -0,0 +1,322 @@
+<?php
+
+class DesignSystemGlobalNavigationModel extends WikiaModel {
+	const DEFAULT_LANG = 'en';
+
+	private $wikiId;
+	private $lang;
+
+	public function __construct( $wikiId, $lang = self::DEFAULT_LANG ) {
+		parent::__construct();
+
+		$this->wikiId = $wikiId;
+		$this->lang = $lang;
+	}
+
+	public function getData() {
+		global $wgUser;
+
+		$data = [
+			'logo' => [
+				'header' => [
+					[
+						'type' => 'link-image',
+						'href' => $this->getHref( 'fandom-logo' ),
+						'image' => 'wds-company-logo-fandom-powered-by-wikia',
+						'title' => [
+							'type' => 'text',
+							'value' => 'Fandom powered by Wikia'
+						]
+					]
+				]
+			],
+			'search' => [
+				'module' => $this->getSearchData()
+			],
+			'create_wiki' => [
+				'links' => [
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "links" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)